CMakeLists.txt

```diff
@@ -165,6 +165,10 @@ if ("${ARCH}" STREQUAL "x86_64")
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mfpmath=sse")
 endif()
 
+if ("${ARCH}" STREQUAL "arm")
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -march=native")
+endif()
```

Comment:
Are you sure this is a good idea?
Can't this break binaries build for redistribution (e.g. in debian packages)?